src/components/fileUpload/resume.tsx

```diff
@@ -62,9 +65,7 @@ const ResumeUpload: React.FC<ResumeUploadProps> = ({ user_id, url }) => {
       const postResponse = await axiosInstance.post(
         '/register/upload-image',
         formData,
-        {
-          headers: { 'Content-Type': 'multipart/form-data' },
-        },
+        { headers: { 'Content-Type': 'multipart/form-data' } },
       );
       const { Location } = postResponse.data.data;
```

Comment:
1. Why is the variable 'Location' named like this (first letter should be small case), project follows camel case
2. Don't blindly trust api to return what you expect from it, there should be a check to see if you are getting response that you are expecting